cmd/expiration-mailer/main.go

```diff
@@ -106,7 +106,7 @@ func (lim *limiter) check(address string) error {
 
 	lim.maybeBumpDay()
 	if lim.counts[address] >= lim.limit {
-		return fmt.Errorf("daily mail limit exceeded for %q", address)
+		return errors.New("daily mail limit exceeded")
```

Comment:
We should specify that it's for a specific address:

```suggestion
		return errors.New("daily mail limit exceeded for this email address")
```